icubam/backoffice/templates/icu.html

```diff
@@ -9,7 +9,7 @@
       <div class="container-fluid">
         <div class="row mb-2">
           <div class="col-sm-6">
-            <h2>{{_("Manage your")}} {{_("ICUs")}}</h2>
+            <h2>{{ _('Manage ICUs') }}</h2>
```

Comment:
I think this pattern occurs in other file. Why not keeping it the way it was ?

Reply:
Hi Olivier thanks for your comments. While rebasing my feature branch to master this conflict happened, I guess decomposing the translations would lead to DRY-er localization files. I'll update the branch.

Reply:
thank you !